backend/LexBoxApi/GraphQL/ProjectMutations.cs

```diff
@@ -127,6 +128,28 @@ public record CreateProjectResponse(Guid? Id, CreateProjectResult Result);
         return dbContext.Projects.Where(p => p.Id == input.ProjectId);
     }
 
+    [Error<NotFoundException>]
+    [Error<DbError>]
+    [Error<RequiredException>]
+    [UseMutationConvention]
+    [UseFirstOrDefault]
+    [UseProjection]
+    public async Task<IQueryable<DraftProject>> ChangeDraftProjectName(ChangeDraftProjectNameInput input,
```

Comment:
I think this method can be removed, including `ChangeDraftProjectNameInput` and the corresponding additions to the gql schema. The same goes for `ChangeDraftProjectDescription`

Reply:
Yep, missed those when I took out PromoteDraftProject. Will do.

Reply:
Done in commit f2630c0a1890c2884a6fc116e63918c76a6c3025.

Reply:
And in commit 77752272b1d38d89b92887e7ce37e6f63bebeb26 where I deleted the ChangeDraftProjectNameInput (and ...Description) input records.

Reply:
You need to update the graphql schema file too.